docs/README.md

```diff
@@ -1,40 +1,105 @@
-# nf-core/coproid: Documentation
+# ![nf-core/coproid](assets/img/coproid_logo.png)
```

Comment:
What is the purpose of this page, that is different from the root README?

Reply:
I would also like to get rid of it, but this is a requirement of the nf-core tools linting.
Deleting it yiels a linting error.

Reply:
Huh,ok. Well in EAGER we just have: https://github.com/nf-core/eager/blob/master/docs/README.md

It's more like a Table of Contents. Maybe it is safer to just have it like eager, then less maintanence required?

Reply:
I keep it there as it is now the landing page of the documentation.